etcdctl/README.md

```diff
@@ -1438,12 +1438,18 @@ RPC: UserRevokeRole
 
 ## Utility commands
 
-### MAKE-MIRROR [options] \<destination\>
```

Comment:
we need to keep the command backward compatibility

Reply:
@xiang90 is right we need to maintain this, we should be able to do both.

Reply:
Can you suggest the CLI options and usage examples then? I’m confused what is required to get this PR merged now as it’s been back and forth quite a bit.

Reply:
I will outline the changes. I proposed a breaking change for 3.4 to clean this up but Xiang is right we should not break backward compatibility. I will write this up with a patchfile no later then tomorrow night. Thanks for your patience and understanding.